### PR TITLE
Add portable criu test for ubuntu and ubi

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Add portable criu test with keep checkpoint and restore functions
- Add images for both ubuntu and ubi
- These tests will only run when jdk supports criu
- Related Issue: https://github.com/eclipse-openj9/openj9/issues/14486 https://github.com/adoptium/aqa-tests/issues/3916

### CODE REVIEW

**Comment by LongyuZhang** [comment]
@llxia @sophia-guo Could you review it? Thanks.

**Comment by LongyuZhang** [comment]
Test Links:
sanity.external
- JDK11: hyc_grinder 26434
- JDK17: hyc_grinder 26435

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -622,6 +624,20 @@ def runTest( ) {
...
 				}
 			}
+
+			// Private Docker Registry Login, temporarily only hyc-jenkins has private Docker Registry
+			if (env.BUILD_URL.contains('hyc')) {
+				if (env.BUILD_LIST.contains('external') && env.DOCKER_REGISTRY_URL != "" && env.DOCKER_REGISTRY_URL_CREDENTIAL_ID != "" ) {	
+					// USERNAME and PASSWORD reference with a withCredentials block will not be visible within job output
+					withCredentials([usernamePassword(credentialsId: "${env.DOCKER_REGISTRY_URL_CREDENTIAL_ID}", usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+						echo "Private Docker Registry login starts:"
+						sh 'echo $PASSWORD | docker login --username=$USERNAME --password-stdin ' + "$env.DOCKER_REGISTRY_URL"
```
[the remark this answers is not in the document]
Comment (LongyuZhang) [reply buildenv/jenkins/JenkinsfileBase]: Docker login has been moved to tests where login is needed (push new image and pull image). New grinder is hyc 26514 and 26515. Thanks.

**Comment by llxia** [comment]
@LongyuZhang I will need to have a desktop review about this PR. I have several questions about this.
- This PR contains two binaries - criu and libcriu.so. This may not be allowed.
- Folder structure

**Comment by LongyuZhang** [comment]
@llxia The two criu binaries have been removed from the commit. They will be downloaded in dockerfile when building the ubi docker image. Test links: hyc_grinder 26609 and 26610. Thanks.

**Comment by llxia** [comment]
I will approve this PR. Other improvements (i.e., use `getDependency` job to pre-download criu and libcriu.so binaries, reorganize the folder structure, etc) can be added in the future PR.

**Review by llxia — APPROVED** [review]

**Review by renfeiw — APPROVED** [review]
LGTM

**Pull request merged**